### ISSUE TITLE
[BUG] Gradle Plugin does not work with Gradle 7.0

### ISSUE THREAD

**Issue by SourcePond** [OpenAPITools/openapi-generator#9253]
##### Description

When using the plugin with Gradle 7.0, the build aborts with following error message:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':microservice-with-api-api:openApiGenerate' (type 'GenerateTask').
  - Type 'GenerateTask' property 'input' annotated with @Internal should not be also annotated with @Input.
    
    Reason: A property is ignored but also has input annotations.
    
    Possible solutions:
      1. Remove the input annotations.
      2. Remove the @Internal annotation.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#ignored_property_must_not_be_annotated for more details about this problem.

* Try:
Run with --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.internal.execution.WorkValidationException: A problem was found with the configuration of task ':microservice-with-api-api:openApiGenerate' (type 'GenerateTask').
  - Type 'GenerateTask' property 'input' annotated with @Internal should not be also annotated with @Input.
    
    Reason: A property is ignored but also has input annotations.
    
    Possible solutions:
      1. Remove the input annotations.
      2. Remove the @Internal annotation.
```     

##### openapi-generator version

5.1.0

**Comment by mattea36** [comment]
There seems to be this PR #9059

**Comment by wing328** [comment]
Closed via #9059
